HPXMLtoOpenStudio/measure.rb

```diff
@@ -4192,6 +4289,38 @@ def self.get_foundation_and_walls_top()
     walls_top = foundation_top + 8.0 * @ncfl_ag
     return foundation_top, walls_top
   end
+  
+  def self.get_location_hierarchy(ba_cz_name)
+    if [Constants.BAZoneHotDry, Constants.BAZoneHotHumid].include? ba_cz_name
+      return [HPXML::LocationGarage,
+              HPXML::LocationLivingSpace,
```

Comment:
No need to have any additional choices beyond HPXML::LocationLivingSpace, every building will have this space type. (Same comment applies to elsif conditions below.)